src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -258,8 +258,7 @@ public void copy() {
         List<BibEntry> selectedEntries = getSelectedEntries();
 
         if (!selectedEntries.isEmpty()) {
-            List<BibtexString> stringConstants = getStringValues();
-
+            List<BibtexString> stringConstants = database.getDatabase().getUsedStrings(selectedEntries).stream().toList();
```

Comment:
Use `getStringValues()`, but modify the method to use `getUsedStrings(selectedEntries)`